controllers/controllers.go

```diff
@@ -198,6 +198,15 @@ func isValidWindowsNode(o client.Object, byoh bool) bool {
 	return true
 }
 
+// getWindowsNodeVersions returns a list of all the WMCO versions that have last configured any nodes
+func getWindowsNodeVersions(nodes []core.Node) []string {
+	versions := []string{}
+	for _, node := range nodes {
+		versions = append(versions, node.Annotations[metadata.VersionAnnotation])
+	}
+	return versions
```

Comment:
opted to return a string slice here instead of a map since it may be useful to see how many nodes have the version annotation, e.g. if len(versions) != len(windoesNodes) then we know a node is currently being configured.